docs/tutorial.rst

```diff
@@ -1123,48 +1123,246 @@ We can then finally obtain the tree sequence::
+
+    interval =  (0.0, 1.0)
+                    26
+           ┏━━━━━━━━┻━━━━━━━┓
+           0                7
+           ┃                ┃
+           25               ┃
+      ┏━━━━┻━━━━┓           ┃
+      23        24          ┃
+    ┏━┻━┓    ┏━━╋━━━┓       ┃
+    ┃   21   ┃  ┃   22      20
+    ┃  ┏┻━┓  ┃  ┃  ┏┻━┓  ┏━━╋━━┓
+    10 14 19 11 18 15 17 12 13 16
+
+    interval =  (1.0, 2.0)
+                      28
+                 ┏━━━━┻━━━━━┓
+                 ┃          27
+                 ┃        ┏━┻━━┓
+                 26       ┃    ┃
+            ┏━━━━┻━━━━┓   ┃    ┃
+            0         7   4    8
+            ┃         ┃  ┏┻━┓  ┃
+            21        ┃  ┃  ┃  ┃
+    ┏━━┳━━┳━┻┳━━┳━━┓  ┃  ┃  ┃  ┃
+    14 19 10 13 16 18 12 15 17 11
+
+The trees have fully coalesced and we've successfully combined a forwards-time
+Wright-Fisher simulation with a coalescent simulation: hooray! However, these
+trees have some slightly odd properties which are important to be aware of.
+In both trees we can see that the old roots are still present in the trees,
+even through they have only one child and are clearly redundant. While this
+redundancy is not important for many tasks, there are some subtle gotchas:
+
+1. When computing statistics on the number of nodes, edges or trees in a tree
+   sequence, having these unary edges and redundant nodes will slightly
+   inflate the values.
+2. If you are computing the overall tree "height" by taking the time of the
+   root node, you may overestimate the height because there is a unary edge
+   above the "real" root (this would happen if one of the trees had already
+   coalesced in the forwards-time simulation).
+
+For these reason it is usually better to remove this redundancy from your
+computed tree sequence. Luckily, this is easily done using the
+:meth:`.TreeSequence.simplify` method:
 
+.. code-block:: python
 
+    final_ts = coalesced_ts.simplify()
 
+    for tree in final_ts.trees():
+        print("interval = ", tree.interval)
+        print(tree.draw(format="unicode"))
 
+giving us::
+
+    interval =  (0.0, 1.0)
+          17
+      ┏━━━┻━━━━┓
+      ┃        15
+      ┃     ┏━━┻━━┓
+      ┃     13    14
+      ┃   ┏━┻┓  ┏━╋━━┓
+      10  ┃  11 ┃ ┃  12
+    ┏━╋━┓ ┃ ┏┻┓ ┃ ┃ ┏┻┓
+    2 3 6 0 4 9 1 8 5 7
+
+    interval =  (1.0, 2.0)
+              19
+        ┏━━━━━┻━━━━━┓
+        ┃           18
+        ┃         ┏━┻┓
+        17        ┃  ┃
+    ┏━━━┻━━┓      ┃  ┃
+    ┃      ┃      ┃  16
+    ┃      ┃      ┃ ┏┻┓
+    ┃      11     ┃ ┃ ┃
+    ┃ ┏━┳━┳┻┳━┳━┓ ┃ ┃ ┃
+    2 4 9 0 3 6 8 1 5 7
+
+This final tree sequence is topologically identical to the original tree sequence,
+but has the redundant nodes and edges removed. Note also that he node IDs have been
+reassigned so that the samples are 0 to 9 --- if you need the IDs from the original
+tree sequence, please set ``map_nodes=True`` when calling ``simplify`` to get a
+mapping between the two sets of IDs.
```

Comment:
Perhaps add here:
```
Why recording the initial generation?
=============================

We can now see why it is essential that the forwards simulator records the *initial* generation in a tree sequence
that will later be used as a ``from_ts`` argument to ``msprime.simulate( )``. In the example above, if node ``7`` was not in the tree sequence, we would not know that the segment that node ``20`` inherits from on ``[0.0, 1.0)`` and the segment that node ``12`` inherits from on ``[1.0, 2.0)`` both exist in the same node (here, node ``7``).

However, note that although the intial generation (above, nodes ``0``, ``4``, ``7``, and ``8``) must be in the tree
sequence, they do *not* have to be samples. The easiest way to do this (as is what is done in ``pyslim``) is to
(a) retain the initial generation as samples throughout the forwards simulation (so they persist through ``simplify()``), but then (b) before recapitation, removing the flags that mark them as samples, so that 
``simulate()`` does not simulate their entire history as well.
```